app/packages/utilities/src/index.ts

```diff
@@ -307,12 +309,29 @@ export const LIST_FIELD = "fiftyone.core.fields.ListField";
 export const JUST_FIELD = "fiftyone.core.fields.Field";
 export const VECTOR_FIELD = "fiftyone.core.fields.VectorField";
 export const DETECTION_FILED = "fiftyone.core.labels.Detection";
-export const KEYPOINT_FILED = "fiftyone.core.labels.Keypoint";
-export const REGRESSION_FILED = "fiftyone.core.labels.Regression";
+export const KEYPOINT_FIELD = "fiftyone.core.labels.Keypoint";
+export const REGRESSION_FIELD = "fiftyone.core.labels.Regression";
 export const GROUP = "fiftyone.core.groups.Group";
 
 export const VALID_LIST_FIELDS = [FRAME_SUPPORT_FIELD, LIST_FIELD];
 
+export const DISABLED_LABEL_FIELDS_VISIBILITY = [
+  DETECTION_FIELD,
+  DETECTIONS_FIELD,
+  CLASSIFICATION_FIELD,
+  CLASSIFICATIONS_FIELD,
+  KEYPOINT_FIELD,
```

Comment:
Should a `KEYPOINTS_FIELD` also be included? i.e. `fiftyone.core.labels.Keypoints`

Reply:
good catch!